### DISCUSSION TITLE
Features results in "Errors: Unexpected token . in JSON at position 503"

### DISCUSSION THREAD

**Discussion by feebb** [rern/rAudio#541] [Fixed Issues]
Love rAudio!! Generally works great, but recently I have got this message "Errors: Unexpected token . in JSON at position 503" when clicking on features. I want to set autoplay to on--is there a way to do this manually?

**Comment by feebb** [comment]
Still not able to see Features on Web interface (using Microsoft Edge on PC) or on local screen. Get the following message:

```
Errors: Unexpected token . in JSON at position 503
{ "page" : "features" , "autoplay" : false , "autoplaycd" : false , "hostname" : "rAudio" , "lcd" : false , "login" : false , "mpdscribble" : false , "mpdscribbleconf" : [ "", "" ] , "streaming" : false , "hostapd" : false , "hostapdconf" : [ "192.168.5.1","raudioap" ] , "ssid" : "rAudio" , "wlanconnect" : true , "shairport-sync" : false , "snapserver" : false , "snapclient" : false , "snapcastconf" : 800 , "spotifyd" : false , "upmpdcli" : false , "localbrowser" : true , "localbrowserconf" : [ 0, ▮.7, "NORMAL", false ] , "smb" : false , "smbconf" : [ true, true ]}
```

**Reply by rern** [reply]
`...position 503` is where the error occured.

Edit file: `/srv/http/data/system/localbrowser.conf` - change `.7` to `0.7`

Next update, save setting will prepend leading 0 if missing.

**Reply by feebb** [reply]
Fixed! Thanks rern!!
